internal/handlers/metric_test.go

```
@@ -241,17 +241,17 @@ func TestBatchUploadsHandler(t *testing.T) {
-		},
+		//{
+		//	name: "Hash Mismatch",
+		//	metrics: []entities.Metrics{
+		//		{ID: "requests", MType: string(entities.CounterMetricName), Delta: intPtr(5)},
+		//	},
+		//	expectedCode: http.StatusBadRequest,
+		//	includeHash:  true,
+		//	correctHash:  false,
+		//	setupSecret:  true,
+		//	description:  "Fails due to hash mismatch when secret is provided",
+		//},
```

golangci-lint (1.62.2)

[error] commentFormatting: put a space between `//` and comment text

(gocritic)